sdk/nodejs/src/cloudClient.ts

```diff
@@ -22,6 +22,15 @@ const castIncomingUser = (user: DVCUser) => {
     return user
 }
 
+const throwIfUserError = (err: AxiosError, isVariableRequst?: boolean) => {
+    const data = err.response?.data as { message: string, statusCode: number }
```

Comment:
no need to define the `statusCode` field here since you aren't using it 

I'd also make the message field optional since you dont know for sure that it's actually there.

Reply:
oh true, fixed!